### DISCUSSION TITLE
My ideas

### DISCUSSION THREAD

**Discussion by Gorilli09** [ikemen-engine/Ikemen-GO#1351] [Feature Request]
# Tournament Mode
It would be cool if there was a Tournament mode with up to 8 players in the engine like in Super Street Fighter II and Virtua Fighter on the 32X.

# Controller buttons instead of numbers in controls option menu
It would be less confusing to add in controller buttons in the controls option menu.

# Customizable D & W Buttons
Gives the D & W Buttons specific functions like simultaneous button presses.

# xBRZ Shaders
xBRZ looks better than HQ.

**Comment by shinra358** [comment]
xbrz is actually the sucessor to hq and xbr. definitely needed. i thought tournament mode was already added. mustve been a fork. 

https://www.youtube.com/watch?v=uut-8Z_KUj8&t=1684s

but i agree with everything in this list.
